### ISSUE TITLE
Merging Hyperparameters for Decoder only LLMs

### ISSUE THREAD

**Issue by pramesh39** [yule-BUAA/MergeLM#44]
Hello, 

Thank you for your great work. Could you share details on how the merging hyperparameters (like scaling co-efficient and pruning factor) were selected for the decoder-only models? While validation data was used for the GLUE tasks, it is unclear what data was used when merging decoder-only models as in Table 1.

**Comment by yule-BUAA** [comment]
Hello Ramesh,

Thank you for your interest in our work. We sample 10% of the data from each benchmark to construct the validation set used for grid search on hyperparameters. This process is applied uniformly to all baseline methods as well as DARE, ensuring a fair and consistent comparison across all approaches.
